README.md

```diff
@@ -434,29 +439,29 @@ ASP.NET Integration
 1. Install the `Sharpbrake.Http.Module` package from NuGet (you can use "Package
    Manager Console" from Visual Studio):
 
-  ```
-  PM> Install-Package Sharpbrake.Http.Module
-  ```
+   ```
+   PM> Install-Package Sharpbrake.Http.Module
+   ```
```

Comment:
They are intentional, the previous indentation doesn't indent properly code snippets in relation to explanations.